components/AgencyBarChart.vue

```diff
@@ -244,6 +248,17 @@ const options: ThisTypedComponentOptionsWithRecordProps<
         )
       })
     }
+  },
+  mounted() {
+    const barChart = this.$refs.barChart as Vue
+    const barElement = barChart.$el
+    const canvas = barElement.querySelector('canvas')
+    const describedbyId = `${this.titleId}-graph`
+
+    if (canvas) {
+      canvas.setAttribute('role', 'img')
+      canvas.setAttribute('aria-labelledby', describedbyId)
```

Comment:
```suggestion
      canvas.setAttribute('aria-labelledby', labelledbyId)
```
こちらもですね。他4つのグラフコンポーネントでも同様ですー。